src/modules/75-cf/components/FlagActivation/hooks/useEditFlagDetailsModal.tsx

```diff
@@ -143,15 +152,28 @@ const useEditFlagDetailsModal = (props: UseEditFlagDetailsModalProps): UseEditFl
                 )}
 
                 <Container>
-                  <Button intent="primary" text={getString('save')} type="submit" />
-                  <Button
-                    minimal
-                    text={getString('cancel')}
-                    onClick={e => {
-                      e.preventDefault()
-                      hideEditDetailsModal()
-                    }}
-                  />
+                  <Layout.Horizontal
+                    height={32}
+                    spacing="small"
+                    style={{ alignItems: 'center' }}
```

Comment:
Instead of an inline style, you can use the `flex` prop
```suggestion
                    flex={{ alignItems: 'center' }}
```

Reply:
Hey @knagurski thanks for the suggestion. If I change the style to flex the buttons appear quite far apart:

<img width="357" alt="image" src="https://user-images.githubusercontent.com/103498167/165273084-4653903f-dabc-4567-9129-c1444d226d0a.png">
` flex={{ alignItems: 'center' }}`

However with the style prop:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/103498167/165273291-152f4756-3919-4f6d-a1b1-9e83f9d693c3.png">
`style={{ alignItems: 'center' }}`

What do you think?

Reply:
Try `flex={{ alignItems: 'center', justifyContent: 'flex-start' }}`.

Actually, just thinking about it, do you need the `align-items` at all? Both buttons are the same size. The `spacing` prop might be all you need.

Reply:
you are spot on actually, only need the spacing prop :) nice!